tools/chpldoc/chpldoc.cpp

```diff
@@ -306,6 +306,8 @@ static void checkKnownAttributes(const AttributeGroup* attrs) {
         // ignore, it's a known attribute
       } else if (name == UniqueString::get(gContext, "chpldoc.attributeSignature")) {
         // ignore, it's a known attribute
+      } else if (name == UniqueString::get(gContext, "chpldoc.hideImplType")) {
```

Comment:
If we have enough of these, I feel like we ought to just define a map then check it. Looks like we have at least 3, might be worth it.